_inc/client/at-a-glance/stats/index.jsx

```diff
@@ -117,7 +117,7 @@ class DashStats extends Component {
 				<div className="jp-at-a-glance__stats-chart">
 					<Chart data={ chartData } barClick={ this.barClick } />
 					{
-						0 < chartData.length ? '' : <Spinner />
+						chartData.length <= 0 && <Spinner />
```

Comment:
Here `.length` would probably never be a negative number.

Reply:
Good catch! Will fix.